fabfile.py

```diff
@@ -15,3 +19,26 @@ def test():
 
 def run():
     local('{}/manage.py runserver 8000'.format(BASE_DIR))
+
+
+def psql(query, no_sudo=False, is_file=False):
+    sudo = 'sudo -u postgres'
+    if no_sudo:
+        sudo = ''
+
+    if is_file:
+        local('{} psql < {}'.format(sudo, query))
+    else:
+        local('{} psql -c "{}"'.format(sudo, query))
+
+
+def setup(*args, **kwargs):
+    no_sudo = True if 'no-sudo' in args else False
+    bootstrap_sql = kwargs['sql'] if 'sql' in kwargs else None
+    psql("DROP DATABASE IF EXISTS mfl", no_sudo)
```

Comment:
For this section, it is more newbie friendly and DRY if you import `django.conf.settings` ( that will require you to add a few lines of Django configuration on top of the fabfile ) and use string interpolation to interpolate in the default database name, database username and database password. 

That will ensure that there is only one place to change database settings.